### DISCUSSION TITLE
Export GPS recording

### DISCUSSION THREAD

**Discussion by Greekmountains** [devemux86/cruiser#674] [Q&A] [answered]
I have been asked by somebody in the OSM community, if I can upload the GPS recorded tracks from my motorcycle offroad trips  using the cruiser app . I know how to record the track. Is it possible afterwards to download the track onto my computer and then upload it to the OSM community.
If this cannot be done can anyone recommend a GPS recording app that would allow me to do this hopefully without a huge battery drain
Thanks

**Comment by devemux86** [comment, marked as answer]
@Greekmountains 

You can use the menu "GPS recording" to export a recorded track (image 1).
Or tap the GPS recording panel to use the same menu.

If you save your recorded tracks in "My collection | Tracks",
you can use the export menu that exists in their screen (image 2).

<img src="https://github.com/user-attachments/assets/12fc512f-fd81-4120-bb84-7d29f3965d42" width="50%"/><img src="https://github.com/user-attachments/assets/2c0e776a-7b4a-47b2-821c-2ecb35393c58" width="50%"/>

**Reply by Greekmountains** [reply]
thank you for the help I will try it out and report back! I heard you are the developer and you are Greek is that correct. If yes greetings and if not greetings anyway!!
